### PR TITLE
General code improvements and fixes

### PR DESCRIPTION
**Pull request by osantana**

### CODE REVIEW

File: pactum/__init__.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from . import fields, verbs  # NOQA
 from .action import Action  # NOQA
 from .api import API  # NOQA
-from .querystring import Querystring # NOQA
+from .querystring import QueryString # NOQA
```
Comment (luizdepra) [inline pactum/__init__.py]: :pray:

File: pactum/fields.py [hunk 2]
```diff
@@ -2,6 +2,16 @@
 
 
 class Field(KeyValueElement):
+    def __init__(self, empty=None, nullable=None, **kwargs):
+        super().__init__(**kwargs)
+
+        if empty is not None:
+            self.empty = empty
```
Comment (luizdepra) [inline pactum/fields.py]: Maybe, just pass `empty` into `super().__init__(...)`. Or there is an intended difference here that I can't see?

Comment (osantana) [reply pactum/fields.py]: Don't know. Let me try.

Comment (osantana) [reply pactum/fields.py]: Yeah! It works! Fixed. 🙂

File: tests/test_fields.py [hunk 3]
```diff
@@ -14,6 +14,16 @@ def test_basic_field():
...
 
 
+def test_basic_field_defaults():
+    field = Field()
+
+    assert field.name == ''
+    assert field.type == Field
+    assert field.required
+    assert not field.nullable
+    assert not field.empty
```
Comment (hartungstenio) [inline tests/test_fields.py]: Maybe assert that the attributes are `False`, and souldn't be `None`

```suggestion
    assert field.required is True
    assert field.nullable is False
    assert field.empty is False
```

File: pactum/fields.py [hunk 4]
```diff
@@ -2,6 +2,13 @@
 
 
 class Field(KeyValueElement):
+    def __init__(self, empty=None, nullable=None, **kwargs):
+        super().__init__(empty=empty, **kwargs)
+
+        if nullable is None:
+            nullable = getattr(self, 'nullable', False)
+        self.nullable = nullable
```
Comment (hartungstenio) [inline pactum/fields.py]: What is the idea here?

```python
class MyField(Field):
    nullable = True

my_field = MyField()
assert my_field.nullable is True
```

If so, what is the expected bahaviour for
```python
class MyField(Field):
    nullable = None
```

It is wrong, but can happen 🤔

Comment (osantana) [reply pactum/fields.py]: Should be `False` but the implementation is wrong. I'll fix it.

**Review by luizdepra — DISMISSED** [review]

File: pactum/fields.py (hunk 4 above)
Comment (osantana) [reply pactum/fields.py]: Reimplemented all KEy/Value 'defaults' handling and added a test for this scenario to block regressions.

**Review by hartungstenio — APPROVED** [review]

**Review by luizdepra — APPROVED** [review]

**Pull request merged**